client/extensions/woocommerce/app/products/product-header.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import StickyPanel from 'components/sticky-panel';
+
+const ProductHeader = ( { onTrash, onDuplicate, onSave, translate } ) => {
+	const trashButton = onTrash &&
+		<Button borderless onClick={ onTrash }><Gridicon icon="trash" /></Button>;
+
+	const duplicateButton = onDuplicate &&
+		<Button onClick={ onDuplicate }>{ translate( 'Duplicate' ) }</Button>;
+
+	const saveButton = onSave &&
+		<Button primary onClick={ onSave }>{ translate( 'Save' ) }</Button>;
+
+	return (
+		<StickyPanel>
+			<Card className="products__header">
+				<span>Breadcrumbs > go > here</span>
+				<div className="products__header-right">
```

Comment:
Instead of a position in the classname, how about function (e.g. products__header-actions) in case we ever want to move this left / center :)